Team6220_PowerPlay/src/main/java/org/firstinspires/ftc/team6220_PowerPlay/BaseTeleOp.java

```diff
@@ -26,27 +25,32 @@ public void driveChassisWithController() {
     }
 
     public void driveGrabberWithController() {
-        if (gamepad2.x) {  // press x to close
+        if (gamepad2.a) {
             driveGrabber(false);
-        } else if (gamepad2.a) {  // press a to open
+        } else if (gamepad2.x) {
             driveGrabber(true);
         }
     }
 
     public void driveSlidesWithController() {
-        motorLVSlides.setPower(-gamepad2.left_stick_y);
-        motorRVSlides.setPower(-gamepad2.left_stick_y);
-
-        if (motorLVSlides.getCurrentPosition() < 0) {
-            motorLVSlides.setPower(0.5);
-            motorRVSlides.setPower(0.5);
-        } else if (motorLVSlides.getCurrentPosition() > 9600) {
-            motorLVSlides.setPower(-0.5);
-            motorRVSlides.setPower(-0.5);
+        if (-gamepad2.left_stick_y < 0 && motorLeftSlides.getCurrentPosition() > 600) {
```

Comment:
You should document the desired behavior of this if statement.

Reply:
In the new code it will be documented

Reply:
You should document this complicated if block now, so others and you can understand what is intended.

Reply:
finished